hip-hcc/.SRCINFO

```diff
@@ -1,19 +1,21 @@
 pkgbase = hip-hcc
 	pkgdesc = Heterogeneous Interface for Portability ROCm
 	pkgver = 3.3.0
-	pkgrel = 5
+	pkgrel = 6
 	url = https://github.com/ROCm-Developer-Tools/HIP
 	arch = x86_64
 	license = MIT
 	makedepends = libelf
 	makedepends = cmake
 	makedepends = python
-	makedepends = hcc
 	makedepends = comgr
 	makedepends = git
 	depends = hsa-rocr
+	depends = hcc
+	depends = rocminfo
 	provides = hip
 	conflicts = hip
+	options = !strip
```

Comment:
Is this really needed? I usually refrain from using !strip, especially when building from source.

Reply:
Actually yes, it is mandatory to pass the reduction test as describe in #135. You may check yourself to be sure that it is not happening only on my system.